spec/cc/engine/content_resolver_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+require "rubocop"
+require "cc/engine/content_resolver"
+
+module CC::Engine
+  describe ContentResolver do
+    cops = RuboCop::Cop::Cop.all
+    whitelist = File.read("./spec/support/docs_whitelist.txt").lines.map(&:chomp)
+
+    it "has cops" do
+      expect(cops.count).to eq 310
```

Comment:
Minor: This might get annoying to assert when we upgrade rubocop. Do we really care about the exact number? If new ones are added & missing content, other specs will fail and we'll need to add them to the whitelist or write content, right?